app/client/ui/CustomSectionConfig.ts

```diff
@@ -558,9 +559,14 @@ export class CustomSectionConfig extends Disposable {
         return cssCustomUrlDetails(
           cssTextInput(
             this._url,
-            async value => this._url.set(value),
+            async (value) => {
+              if ((!value.length || await userTrustsCustomWidget())) {
```

Comment:
Too many ((, and shouldn't we check the validity here?

Reply:
Yeah I originally wanted to use the native `reportValidity` but for some reason I don't understand, it always returns true here… so the user doesn't get the native form validation with native browser message, unlike in the custom widgets gallery where it works.

So yeah I didn't use it. Sometimes those native apis are only usable if the browser detects we actually are in a user interaction, maybe something like that is the culprit. Really I'm surely missing something but I can't figure out what.

Reply:
Sorry for that. This is caused by the `rawTextInput` component in `editableLabel.ts`. It tries to be clever and disable the input before saving, so the reportValidity doesn't work. I'll file an issue for it.